source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj

```
@@ -23,7 +23,7 @@
     <Title>Cake.ExtendedNuGet</Title>
     <Description>Cake Build addon to provide Aliases for NuGet package inspection.</Description>
     <Summary>Cake build aliases for NuGet package inspection.</Summary>
-    <PackageTags>Cake Script Build</PackageTags>
+    <PackageTags>cake, script, build, cake-build, addin, cake-contrib, cake-addin, nuget</PackageTags>
```

```suggestion
    <PackageTags>cake;script;build;cake-build;addin;cake-contrib;cake-addin;nuget</PackageTags>
```

[Tags must be either space or semicolon-delimited](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#important-nuget-package-metadata)

---

Does it matter? I feel I have "more" done using the ", " as delimiter 😲

---

Hm. I guess NuGet gallery is prepared for someone like me: Using ", " works, despite the specs.
I wonder whether we should check that in CakeContrib.Guidelines, too.

---

I suspect using `,` as separator might work (undocumented) or maybe it'll put everything as one giant tag :smile:

Some interesting comments here: https://github.com/dotnet/docs/issues/13553#issuecomment-518444966